.github/scripts/print_pr_details.py

```diff
@@ -0,0 +1,55 @@
+# .github/scripts/print_pr_details.py
+from github import Github
+import os
+
+# Initialize the GitHub client
+token = os.getenv('GITHUB_TOKEN')
+g = Github(token)
+
+# Get repository information from environment variables
```

Comment:
The script should handle cases where environment variables are missing by adding error handling for GITHUB_TOKEN and GITHUB_REPOSITORY